src/main/java/org/springframework/retry/backoff/ExponentialBackOffPolicy.java

```diff
@@ -172,7 +172,12 @@ public void initialIntervalSupplier(Supplier<Long> initialIntervalSupplier) {
 	 */
 	public void multiplierSupplier(Supplier<Double> multiplierSupplier) {
 		Assert.notNull(multiplierSupplier, "'multiplierSupplier' cannot be null");
-		this.multiplierSupplier = multiplierSupplier;
+		if (multiplierSupplier.get() > 1.0) {
```

Comment:
Same problem here: it is a point of `multiplierSupplier` to evaluate it at runtime for every call.

Reply:
This is reverted now

Reply:
I have a convenient rule: if you agree with me, there is no need to comment `Fixed`, `Done` etc.
I'll see all the requested changes in the next commit you push.
This way I would receive only a single email from this PR knowing that everything requested is there.
It is totally fine to discuss if you are disagree or I am wrong.

Only exception is to notify explicitly if PR was rebased. In this case GitHub doesn't sent an email 🤷